doc/nrf/libraries/nfc/lib_nfc_ndef.rst

```diff
@@ -15,17 +15,17 @@ If you want to implement your own polling device, you can use the supplied parse
 
 Currently, the following NDEF libraries are supported:
```

Comment:
This sentence will not be needed with the planned updates, I think.

Reply:
fixed